src/pages/Lend/LoanChannelCategoryControl.vue

```diff
@@ -381,12 +385,19 @@ export default {
 		this.updateFromParams(this.pageQuery);
 
 		// Prevent pop-in by loading data from the Apollo cache manually here instead of just using the subscription
-		const baseData = getCachedChannel(
-			this.apollo,
-			this.loanChannelQueryMap,
-			this.targetedLoanChannelURL,
-			this.loanQueryVars
-		);
+		const baseData = this.enableQuickFilters
+			? getFilteredLoanChannel(
```

Comment:
Is there a situation where the user would be able to get this filtered channel from the cache? In this current setup, it will always be a slightly slowly, since `fetchLoanChannel` returns a promise. On the same note, does this need an `await` before it?